lib/ansible/modules/cloud/google/gcp_bigquery_dataset_info.py

```diff
@@ -29,10 +29,11 @@
 
 DOCUMENTATION = '''
 ---
-module: gcp_bigquery_dataset_facts
+module: gcp_bigquery_dataset_info
 description:
-- Gather facts for GCP Dataset
-short_description: Gather facts for GCP Dataset
+- Gather info for GCP Dataset
+- This module was called C({{ old_name }}) before Ansible 2.9. The usage has not changed.
```

Comment:
```suggestion
- This module was called C(gcp_bigquery_dataset_facts) before Ansible 2.9. The usage has not changed.
```
Sorry for the confusion; we used `{{ old_name }}` to indicate the old name of the module, assuming that you would replace it while updating the modules. Ansible will simply typeset this as-is and the user will read `{{ old_name }}` and not the real old name of the module.

Reply:
Ah, got it! I assumed there was some Jinja magic happening somewhere. Fix made!